packages/blockchain-wallet-v4-frontend/src/modals/SfoxExchangeData/Create/AcceptTerms.js

```diff
@@ -92,7 +96,7 @@ class AcceptTerms extends Component {
     const busy = this.props.errorStatus.cata({
       Success: () => false,
       Failure: (err) => err,
-      Loading: () => true,
+      Loading: () => Remote.Loading,
       NotAsked: () => false
     })
```

Comment:
I'm not sure this makes much sense. The cata is not really for the errorStatus of signup. Error status is one of the four possibilities. It's for sfoxSignup. i.e signing up for sfox could be in a state of success (not busy, no message), failure (not busy, error message), loading (busy, no message, notasked (not busy, no message). If you need error and busy maybe something like this would work:

```
const { busy, err } = this.props.errorStatus.cata({
      Success: () => ({ busy: false }),
      Failure: (err) => ({ busy: false, err }),
      Loading: () => ({ busy: true }),
      NotAsked: () => ({ busy: false })
    })
```

Reply:
Ah I like this, thanks. Change made.

Reply:
Nice, this reads much better now. Thanks!